docs/USING_GRID_REDUCERS.md

```diff
@@ -109,15 +109,15 @@ If you want to hide grid reducers from the root of your state:
 
 ````js
 import { combineReducers } from 'redux';
-import { GridRootReducer } from 'react-redux-grid';
+import { rootReducer } from 'react-redux-grid';
```

Comment:
You're absolutely right :). I've changed it to
```js
import { Reducers as rootReducer } from 'react-redux-grid';
```
to clarify that `Reducers` is actually the rootReducer